c7n/resources/backup.py

```diff
@@ -87,3 +91,66 @@ def get_resources(self, resource_ids, cache=True):
 class KmsFilter(KmsRelatedFilter):
 
     RelatedIdsExpression = 'EncryptionKeyArn'
+
+
+@BackupVault.action_registry.register('modify-policy')
+class ModifyPolicyStatement(ModifyPolicyBase):
+    """Action to modify Backup Vault IAM policy statements.
+
+    :example:
+
+    .. code-block:: yaml
+
+           policies:
+              - name: backup-vault-get-policy
+                resource: backup-vault
+                filters:
+                  - type: value
+                    key: BackupVaultName
+                    value: "backup_vault_name"
+                actions:
+                  - type: modify-policy
+                    add-statements:
+                      - "Sid": "AddMe"
+                        "Effect": "Allow"
+                        "Principal": {"AWS": "arn:aws:iam::116249476610:root"}
+                        "Action": ["backup:GetBackupVaultAccessPolicy"]
+                        "Resource": "*"
+                    remove-statements: []
+    """
+    permissions = ('backup:PutBackupVaultAccessPolicy', 'backup:GetBackupVaultAccessPolicy')
+
+    def process(self, resources):
+        results = []
+        client = local_session(self.manager.session_factory).client('backup')
+
+        for r in resources:
+            policy = json.loads(
+                client.get_backup_vault_access_policy(
+                    BackupVaultName=r['BackupVaultName']
+                )['Policy']
+            )
+            policy_statements = policy.setdefault('Statement', [])
+
+            new_policy, removed = self.remove_statements(
+                policy_statements, r, CrossAccountAccessFilter.annotation_key)
+            if new_policy is None:
+                new_policy = policy_statements
+            new_policy, added = self.add_statements(new_policy)
+
+            if not removed and not added:
+                continue
+
+            results += {
+                'Name': r['BackupVaultName'],
+                'State': 'PolicyModified',
+                'Statements': new_policy
```

Comment:
whats the use case on the action return value?

Reply:
Ah yeah that's not needed